release-notes-3.4.0.md

```diff
@@ -3,10 +3,8 @@ ${version-number}
 
 #### New Features
 - **SCMOD-7343**: Audit Service has been updated to support Elasticsearch 7.2.0
-- **SCMOD-4930**: Removed elastic search cluster name specification
```

Comment:
I asked you to check on this, not simply remove it. If it is the case that users can no longer define the cluster name of ES for the audit service to connect to surely this is something that needs called out.

Reply:
Leave 4930 in the release notes, it won't hurt.

Reply:
I'm absolutely fine with it being there its just that the description does not help a user to know what the change is. Can we re-word it a bit to better reflect the change that was made where you no longer can/need to define the ES cluster name?

Reply:
**SCMOD-4930**: Removed elastic search cluster name specificatio. The audit service now uses the Elastic HTTP client which does not require a cluster name.